.eslintrc.js

```diff
@@ -1,33 +1,55 @@
 // https://eslint.org/docs/user-guide/configuring
-
 module.exports = {
   root: true,
   parserOptions: {
-    parser: 'babel-eslint',
+    parser: '@babel/eslint-parser',
```

Comment:
ESLint no longer used `babel-eslint` as they will no longer be maintained in the future. https://babel.dev/blog/2020/07/13/the-state-of-babel-eslint